packages/fxa-content-server/app/scripts/views/support.js

```diff
@@ -137,6 +159,11 @@ const SupportView = BaseView.extend({
     });
   },
 
+  navigateOnCancel(e) {
+    e.preventDefault();
```

Comment:
The cancel button never worked correctly(?!) because we didn't do this.  A functional test is added in the PR for this also.